ts/packages/cli/src/commands/export.ts

```diff
@@ -41,6 +41,10 @@ export default class Export extends BaseCommand<typeof Export> {
       description: `Pipe eval commands result to export command to pass \`Configs\` as environment variables to a child-process`,
       command: `<%= config.bin %> eval ... | <%= config.bin %> <%= command.id %> --run 'node index.js'`,
     },
+    {
```

Comment:
Not entirely sure why but this addition does not show up when I run `--help`:
![image](https://github.com/configu/configu/assets/47181182/82627551-84d2-41e3-bce6-b37006983b53)

We should check if this does show up when we create a staging build.